docs/source/whats-new.md

```diff
@@ -4,6 +4,47 @@ What's new
 See [GitHub releases page](https://github.com/OSOceanAcoustics/echopype/releases) for the complete history.
 
 
+# v0.8.2 (2023 November 20)
+
+## Overview
+
+This release includes a number of important performance enhancements, bug fixes, and under-the-hood refactoring to improve code readability.
+
+## Enhancements
+- Full refactor of the functionality to directly storing parsed data to zarr before set_groups_* (#1185, #1218 [NOT MERGED], #1070)
+  - This avoids memory expansion during NaN padding across channel
+- Improve `compute_MVBS` using flox (#1124)
+- Revive and improve `compute_NASC` using flox (#1167)
+- Refactor AZFP XML parser
+  - Parse more parameters and store in the resulting `EchoData` objects (#1135)
+  - Add support for AZFP multiple phase settings (#1182)
```

Comment:
Based on your email exchange with Steve, it sounded like the support that was added would need additional work, right? If so, maybe add some qualifiers to this statement ("initial support", "partial support", ...)